includes/Dashboard.php

```diff
@@ -151,7 +151,7 @@ public function load_stories_dashboard() {
 		// TODO Preload templates.
 		$preload_paths = [
 			'/wp/v2/settings',
-			'/web-stories/v1/web-story?context=edit&order=desc&orderby=modified&page=1&per_page=24&status=publish%2Cdraft&_web_stories_envelope=true',
+			'/web-stories/v1/web-story?embed=author&context=edit&order=desc&orderby=modified&page=1&per_page=24&status=publish%2Cdraft&_web_stories_envelope=true',
```

Comment:
Actually, found a second issue

```suggestion
			'/web-stories/v1/web-story?_embed=author&context=edit&order=desc&orderby=modified&page=1&per_page=24&status=publish%2Cdraft%2Cfuture&_web_stories_envelope=true',
```

The preloading did not work because of the different `status` param.

So two things to fix:

* `_embed`
* `status`